detectron/modeling/FPN.py

```diff
@@ -293,7 +293,7 @@ def add_topdown_lateral_module(
             bias_init=const_fill(0.0)
         )
     # Top-down 2x upsampling
-    td = model.net.UpsampleNearest(fpn_top, fpn_bottom + '_topdown', scale=2)
+    td = model.net.ResizeNearest(fpn_top, fpn_bottom + '_topdown', width_scale=2., height_scale=2.)
```

Comment:
Are these ops exactly equivalent? Please verify that this change does not impact the AP of FPN model zoo models and does not impact the results when training new models.

Reply:
@rbgirshick Thanks for your review. I will check it in several days.